src/utils/events/index.ts

```diff
@@ -270,6 +284,44 @@ const getCreateDomainActionValues = async (
   return domainAction;
 };
 
+const getColonyEditActionValues = async (
+  processedEvents: ProcessedEvent[],
+): Promise<Partial<ActionValues>> => {
+  const mintTokensEvent = processedEvents.find(
```

Comment:
why this name? 🤔

Reply:
I guess I copy-pasted it from somewhere :)

I'll change it.